purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1598,26 +1654,18 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         if (billing.isConnected()) {
             log(LogIntent.DEBUG, PurchaseStrings.UPDATING_PENDING_PURCHASE_QUEUE)
             dispatcher.enqueue({
-                val queryActiveSubscriptionsResult =
```

Comment:
`BillingAbstract.queryPurchases` now returns both active subscriptions and unconsumed in apps so this logic has been moved to the BillingWrapper. We don't need to distinguish between both.